<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEmployeeDialog</class>
 <widget class="QDialog" name="EditEmployeeDialog">
  <property name="windowTitle">
   <string>Редактировать сотрудника</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelFirstName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLastName">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lastNameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelMiddleName">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="middleNameEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>Почта</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelRole">
       <property name="text">
        <string>Роль</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="roleCombo">
       <item>
        <property name="text">
         <string>распределитель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>работник</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
